lambdas/process-journey-event/src/main/resources/statemachine/journey-maps/ipv-core-main-journey.yaml

```diff
@@ -477,3 +481,52 @@ MITIGATION_01_CRI_DCMAW:
     fail-with-no-ci:
       targetState: PYI_ANOTHER_WAY
 
+# Mitigation journey (02)
+MITIGATION_02:
+  response:
+    type: page
+    pageId: page-ipv-identity-document-start
```

Comment:
The first page on the same session mitigation journey is `pyi-suggest-other-options-no-f2f` which we're waiting for - see the dependency on PYIC-3417 (and the associated figma ucd). This new page has 2 options, the first to proceed to the App, and the other to proceed to proving identity another way.
We'll need to add in a new state (`MITIGATION_02_OPTIONS`) as the entry point for the mitigation and that would `next` to what is currently `MITIGATION_02` (though this would then merit a better name).
Let's work through this together.

Reply:
I had considered going through the existing page because new page was'nt ready yet. The reason was to able to test my changes. Once the page is ready., I was planning to change it with another task. I have updated the PR with Draft as you requested  and adjusted it according to the page that will be ready.